common/entity/ParticipationCertificate.ts

```diff
@@ -0,0 +1,52 @@
+import {
+    Entity,
+    Column,
+    PrimaryGeneratedColumn,
+    ManyToOne,
+    JoinColumn,
+    Index,
+    CreateDateColumn
+} from "typeorm";
+import { Student } from "./Student";
+import { Pupil } from "./Pupil";
+
+@Entity()
+export class ParticipationCertificate {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Index({ unique: true })
+    @Column()
+    uuid: string;
+
+    @Column()
+    subjects: string;
+
+    @Column()
+    categories: string;
+
+    @CreateDateColumn({ type: "timestamp" })
+    certificateDate: Date;
+
+    @CreateDateColumn({ type: "timestamp" })
+    startDate: Date;
+
+    @CreateDateColumn({ type: "timestamp" })
+    endDate: Date;
+
+    @Column()
+    hoursPerWeek: number;
+
+    @Column()
+    hoursTotal: number;
+
+    @ManyToOne((type) => Student)
+    @JoinColumn()
+    student: Student;
+
+    @ManyToOne((type) => Pupil)
+    @JoinColumn()
+    pupil: Pupil;
+
```

Comment:
The `medium` is missing here. See this from the certificate generation function:  https://github.com/corona-school/backend/blob/308bbcc8f78dcb973e96be883f183f7b0782e67b/web/controllers/certificateController/index.ts#L189 
It should be also stored in the database when creating a certificate. And of course it should be displayed on the certificate verification page as well.

Reply:
added column [https://github.com/corona-school/backend/commit/442838bde60b2004c7362ce80a02293802544f9b](https://github.com/corona-school/backend/commit/442838bde60b2004c7362ce80a02293802544f9b)